index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kick UUIDs</title>
+    <script src="https://cdn.jsdelivr.net/npm/protobufjs@7.2.5/dist/protobuf.min.js"></script>
+</head>
+<body>
+    <h1>UUID as a Service</h1>
+    <p>No warranty, nothing. Use at your own risk. Good luck! :-)</p>
+    <input type="text" id="ipInput" placeholder="Enter IP">
+    <input type="text" id="pinInput" placeholder="Pin">
```

Comment:
Can you add info on what the Pin is exactly? And where it can be found?

Reply:
Yes, sure, I'll add it directly in the HTML.

Reply:
(doing this in a separate commit)